examples/network_filters.yaml

```diff
@@ -25,3 +25,8 @@ spec:
     services: # filter on Kubernetes services; this will correctly handle the port differences in node vs. pod-level disruptions
       - name: demo # service name
         namespace: chaos-demo # service namespace
+        ports: # optional. List of affected ports. No list means all ports are affected
+          - port: 8080
+            name: regular # optional. Name of the port, used to identify the port affected
```

Comment:
Per my understanding, user can't specify only the name correct?
In a Kubernetes definition, there is a nice feature for `targetPort`:
* https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports

You can provide either a string or a number. If you provide a string Kubernetes will target the named port and you don't have to provide an int value.

Is this something we could /should do?
Hence probably removing "name" and renaming "port" to "servicePort" (I nearly suggested targetPort but it could be confusing in the end)

WDYT?